core/src/main/scala/com/evolutiongaming/kafka/flow/TopicFlow.scala

```diff
@@ -80,6 +80,9 @@ object TopicFlow {
       def apply(records: ConsRecords) = {
         for {
           partitions <- cache.values
+          _ <- Log[F].debug(
+            s"got ConsRecords with partitions ${records.values.keySet}; cached partitions: ${cache.values}"
```

Comment:
I think `cache.values` returns an `F[...]` here so there's not much sense in logging it. On line 82 you already have `partitions` and it's a `Map[Partition, PartitionFlow[F]]`. Probably you'll need only keys from that.  

We could also print information about received consumer records in some pretty way, let's say, "${partition} - ${offset of the last message for it}". Since `ConsumerRecords` is a wrapper around `Map[TopicPartition, Nel[ConsumerRecord[K, V]]]`, that shouldn't be an issue, I suppose. Please don't forget to create this string only inside `Log[F].debug {}` block so that we don't allocate a redundant String when debug logging is disabled